### ISSUE TITLE
"ko" appears to be a copy/paste of "kon"

### ISSUE THREAD

**Issue by emmaly** [jProgr/TokiPonaDictionary#3]
It appears that `ko` is wrong.  It shows the `modifier` part of speech as the same as `kon`.  According to http://tokipona.net/tp/janpije/dictionary.php (and other places, but I don't recall which), `ko` is a noun that means `clay, clinging form, dough, semi-solid, paste, powder`, which is what I had understood it to be.

```
  {
    "word":"ko",
    "meanings":[
      [
        "mod",
        "air-like, ethereal, gaseous"
      ]
    ]
  },
  {
    "word":"kon",
    "meanings":[
      [
        "n",
        "air, wind, odour, soul"
      ],
      [
        "mod",
        "air-like, ethereal, gaseous"
      ]
    ]
  },
```

![Screenshot 2019-11-03 at 7 48 09 PM](https://user-images.githubusercontent.com/89385/68098676-e26c8380-fe72-11e9-9b4f-0a64aaa2718c.png)

**Comment by jProgr** [comment]
Thanks for pointing it out. The meaning of “ko” will be changed as it appears in Toki Pona - The Simplest Language in the World by Robert Warnke (2019):

> noun: semi-solid or squishy substance; clay, dough, glue, paste, powder, gum
> verb transitive: to squash, to pulverize

**Comment by jProgr** [comment]
Fixed by `5f2c41466f81d5271c9e0ec41720f0c917cd5924`.
